src/components/login/Login.js

```diff
@@ -1,43 +1,63 @@
-import React, { useState } from "react";
-import axios from "axios";
-import { signInWithPopup, GoogleAuthProvider } from "firebase/auth";
+import React, { useEffect, useState } from "react";
 import { useDispatch } from "react-redux";
 import { useNavigate } from "react-router-dom";
-import { useQuery } from "react-query";
-import styled from "styled-components";
 import GoogleButton from "react-google-button";
+import { signInWithPopup, GoogleAuthProvider } from "firebase/auth";
+import styled from "styled-components";
+import axios from "axios";
 
 import { signUpActions } from "../../features/signUpSlice";
 import { authentication } from "../../utils/firebase";
+import { userActions } from "../../features/userSlice";
+import { ERROR_MESSAGE, RESPONSE_MESSAGE } from "../../constants";
 
 const Login = () => {
   const navigate = useNavigate();
   const dispatch = useDispatch();
   const [userEmail, setUserEmail] = useState("");
   const [photoURL, setPhotoURL] = useState("");
```

Comment:
성훈님, 아까 photoURL이 제대로 상태 저장이 안되었잖아요-

```suggestion
  const [userInfo, setUserInfo] = useState({
    email: "",
    photoURL: "",
  });

  const signInWithGoogle = async () => {
    const provider = new GoogleAuthProvider();
    const userInfo = await signInWithPopup(authentication, provider);
    const { email, photoURL } = userInfo.user;

    setUserInfo({
      email,
      photoURL,
    });
  };
```

그래서 email이랑 photoURL 상태를 하나의 객체로 묶어서 한 번에 관리하면 어떨까 싶은데,,
의견 부탁드려요!!

Reply:
네 좋은 생각이네요 수정하겠습니다~

Reply:
혹시 이 경우도 const { photoURL, email } 이렇게 photoURL 먼저 해줘야 가능한 건 아니겠죠...ㅎㅎ  
궁금합니다, 확인 후 공유 부탁드려요~

Reply:
잘 받네요 서영님 ㅎㅎ